### PR TITLE
adjust TEF shelf landing on desktop (bug 1145654)

### PR DESCRIPTION
**Pull request by spasovski**
Yay one-offs!

### CODE REVIEW

File: src/media/css/feed/shelves.styl [hunk 1]
```diff
@@ -121,3 +121,26 @@ shelf_curve_color($color) {
...
+    [data-shelf-landing-carrier="telefonica"] {
+        .feed-landing-header {
+            padding: 0;
+        }
+        .feed-landing-curve {
+            border-radius: 0;
+            padding: 40px 0;
+        }
+        .has-desc.feed-landing-header-inner {
+            flex-stack();
```
Comment (ngokevin) [inline src/media/css/feed/shelves.styl]: i *just* merged autoprefixer so you don't have to use the mixin anymore.

File: src/media/css/feed/shelves.styl [hunk 2]
```diff
@@ -121,3 +121,26 @@ shelf_curve_color($color) {
...
+            flex-vertical-center();
```
Comment (ngokevin) [inline src/media/css/feed/shelves.styl]: and i'll be getting rid of this mixin too. so we can just use the rules we'd normally use for flex vertical centering

File: src/media/css/feed/shelves.styl [hunk 3]
```diff
@@ -121,3 +121,26 @@ shelf_curve_color($color) {
...
+            margin: 0;
+        }
+        .feed-landing-header-inner h1 {
+            width: 320px;
```
Comment (ngokevin) [inline src/media/css/feed/shelves.styl]: we have a var for this feed-column-width

File: src/templates/_macros/feed_item.html [hunk 4]
```diff
@@ -225,7 +225,7 @@ <h1 class="feed-item-name">{{ coll.name|translate(coll)|safe }}</h1>
                  src="{{ feed_item.background_image_landing or feed_item.background_image }}"
                  alt="" draggable="false" itemprop="contentURL">
           {% endif %}
-          <div class="feed-landing-header-inner">
+          <div class="feed-landing-header-inner {{ 'has-desc' if feed_item.description }}">
```
Comment (ngokevin) [inline src/templates/_macros/feed_item.html]: make this a property of the .feed-landing-wrapper where all the other stuff is and prefix the class name

**Comment by spasovski** [comment]
Anything else? Sorry about the rush it's meant for tomorrow's push.

**Comment by ngokevin** [comment]
Sorry, let me apply it now and r+. I'm on push duty so I'll make sure it gets in

**Comment by ngokevin** [comment]
it's a bit off-center, but obv. who cares. r+

**Pull request merged**